server.js

```diff
@@ -1,25 +1,19 @@
-app.put('/slambook/:id', async (req, res) => {
+app.delete('/slambook/:id', async (req, res) => {
   const entryId = req.params.id;

```

Comment:
The variable 'entryId' is not defined. It should be obtained from 'req.params.id' instead.